caffe2/CMakeLists.txt

```diff
@@ -824,6 +824,21 @@ if(BUILD_CAFFE2 AND NOT MSVC)
   target_compile_options(torch_cpu PRIVATE "-Wno-sign-compare")
 endif()
 
+if("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
+  target_compile_options_if_supported(torch_cpu "-Wmissing-prototypes")
+  target_compile_options_if_supported(torch_cpu "-Werror=missing-prototypes")
+  set(ignored_files "${CMAKE_CURRENT_SOURCE_DIR}/utils/proto_wrap.cc")
+  foreach(generated_file IN LISTS GENERATED_CXX_TORCH ignored_files)
+    set_source_files_properties(${generated_file} PROPERTIES COMPILE_OPTIONS "-Wno-missing-prototypes;-Wno-error=missing-prototypes")
```

Comment:
Is the plan to fix these in a follow up? Or is it the expected end state?

Reply:
No plan to fix them now.